testapp/build.gradle

```diff
@@ -83,7 +83,7 @@ android {
 
     buildTypes {
         debug {
-            applicationIdSuffix '.debug'
+            //applicationIdSuffix '.debug'
```

Comment:
Do we need remove this ?

Reply:
yes, other than this purchase will not initiate because the package id registered in google play console is without the `.debug1` suffix.